src/server/routes/index.ts

```diff
@@ -1,6 +1,7 @@
 import { Path } from '../../types';
 import ping from './ping';
 import sync from './sync';
+import viewer from './viewer';
 
 const routes: Path[] = [
```

Comment:
Does the ping path need to be added back here? Its not needed but most APIs have one just as a quick health check.